### PR TITLE
[CS2113T-W17-3]-valerietyj-B-ViewSchedules

### PR DESCRIPTION
**Pull request by valerietyj**
Implement B-ViewSchedules

### CODE REVIEW

**Comment by shamussy** [comment]
Please check your code to ensure you do not violate checkstyle violations and commit again thanks

File: src/main/java/duke/task/TaskList.java [hunk 1]
```diff
@@ -167,4 +168,25 @@ public void delete(int i) throws IndexOutOfBoundsException {
         }
         return output;
     }
+
+    /**
+     * Views task and schedule based on a specific date.
+     * @param date Date to be searched for in the form of DD/MM/YYYY
+     * @return An ArrayList of strings representing the list of tasks based on date provided
+     */
+    public ArrayList<Task> vfilter(LocalDate date) {
```
Comment (BrianTKM) [inline src/main/java/duke/task/TaskList.java]: Please rename the method to be more descriptive, such that it is easy to know what the method does just by reading the name.

Comment (BrianTKM) [inline src/main/java/duke/task/TaskList.java]: Please rename to a more descriptive name, such that just by looking at the method name, people know what it is suppose to do

File: src/main/java/duke/task/TaskList.java [hunk 2]
```diff
@@ -167,4 +168,25 @@ public void delete(int i) throws IndexOutOfBoundsException {
         }
         return output;
     }
+
+    /**
+     * Views task and schedule based on a specific date.
+     * @param date Date to be searched for in the form of DD/MM/YYYY
```
Comment (BrianTKM) [inline src/main/java/duke/task/TaskList.java]: Please follow java docs standard

File: src/main/java/duke/task/Deadline.java [hunk 3]
```diff
@@ -3,7 +3,7 @@
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
-
+import java.time.LocalDate;
 import duke.exception.DukeException;
```
Comment (BrianTKM) [inline src/main/java/duke/task/Deadline.java]: Please follow java coding standard

**Comment by shamussy** [comment]
Helped @valerietyj with the merge conflicts

**Comment by shamussy** [comment]
There are still a few minor checkstyle errors,
KIV: 
src/main/java/duke/task/TaskList.java
Line 53:
int dateLength = Integer.parseInt(dateSplit[1].trim());

Have to merge and fix locally, unable to hold others back.

**Pull request merged**